Documentation/gettingstarted/cni-chaining-aws-cni.rst

```diff
@@ -24,7 +24,12 @@ and provide encryption.
 
    Please ensure that you are running version `1.7.9 <https://github.com/aws/amazon-vpc-cni-k8s/releases/tag/v1.7.9>`_
    or newer of the AWS VPC CNI plugin to guarantee compatibility with Cilium.
-   The official upgrade instructions can be found `here <https://docs.aws.amazon.com/eks/latest/userguide/cni-upgrades.html>`_.
+
+   .. code-block:: bash
```

Comment:
```suggestion
   .. code-block:: shell-session
```

Reply:
Just a bit further down there's `code-block:: bash`. So what's the difference? Do we need to replace all of them?

Reply:
`bash` should be for bash scripts, `shell-session` is better suited to commands to run in a terminal (the prompt will be coloured properly, for example, and a `#` would not be interpreted as a comment marker). Joe and I have been trying to enforce the use of `shell-session` everywhere relevant for the new directives (or when edits touch at the existing ones), but we haven't gone through a massive replacement of (most of) the existing `bash` occurrences so far. Could be an option.

Reply:
Thanks for the explanation! I'll change it.

> but we haven't gone through a massive replacement of (most of) the existing bash occurrences so far. Could be an option.

I think it's worth it. Otherwise we'll just keep copying them around :grimacing: